extensions.html

```diff
@@ -397,6 +508,7 @@ <h2>Partial <dfn>Gamepad</dfn> Interface</h2>
           readonly attribute GamepadHand hand;
           readonly attribute FrozenArray&lt;GamepadHapticActuator> hapticActuators;
           readonly attribute GamepadPose? pose;
+          readonly attribute FrozenArray&lt;GamepadLightIndicator>? lightIndicators;
```

Comment:
Is it usual for there to be multiple light indicators?

Reply:
Not for Playstation controllers, but Xbox 360, Wiimote, Joy-Cons, etc have four player indicator LEDs.